internal/cli/opsmanager/admin/backup/blockstore/description.go

```diff
@@ -14,6 +14,7 @@
 package blockstore
 
 const (
-	short = "Manage backup blockstores."
-	list  = "List backup blockstores."
+	short    = "Manage backup blockstores."
+	list     = "List backup blockstores."
+	describe = "Get a backup blockstore."
```

Comment:
maybe we should add the word configuration here, as in, "Manage backup blockstore configuration"